src/algorithms/ctmrg.jl

```diff
@@ -28,16 +28,25 @@ end
 """
     CTMRG(; tol=Defaults.ctmrg_tol, maxiter=Defaults.ctmrg_maxiter,
           miniter=Defaults.ctmrg_miniter, verbosity=0,
-          svd_alg=TensorKit.SVD(), trscheme=FixedSpaceTruncation())
+          svd_alg=TensorKit.SVD(), trscheme=FixedSpaceTruncation(),
+          ctmrgscheme=:AllSides)
```

Comment:
In general the new scheme leads to much faster differentiation since you don't have to differentiate the gauge fixing anymore. In that way I think it's the better default.

On the other hand, it seems to have problems with the Z2 gauge fixing tests with unit cells, so I don't know whether there is a more general problem at hand or not.